tox-envlist is not needed to run tests.
tox version pin is too strict.

--- apeye-1.4.1/tox.ini.orig
+++ apeye-1.4.1/tox.ini
@@ -35,8 +35,6 @@
 isolated_build = True
 requires =
     pip>=21,!=22.2
-    tox-envlist>=0.2.1
-    tox~=3.0
     virtualenv!=20.16.0
 
 [envlists]
